server/spec/test_external_resources_file.rb

```diff
@@ -54,12 +54,12 @@ def test_list_resources_text
   end
 
   def test_get_resources_text
-    body = @erf.get_resource "text","howdy.txt"
+    body = @erf.get_resource "text","1.txt"
     assert body.length > 10 ,"get resource from text directory"
   end
 
   def test_get_resources_image
-    body = @erf.get_resource "image","blue.png"
+    body = @erf.get_resource "image",'UMIngallMallFountan_3253_5.jpg'
```

Comment:
Since the test file is only used during development and changes don't require a release I don't see a need to make the file names configurable.